tests/unit/aws/test_access.py

```diff
@@ -288,42 +295,46 @@ def test_create_uber_principal_no_storage(mock_ws, mock_installation, locations)
 
 def test_create_uc_role_single(mock_ws, installation_single_role, backend, locations):
     aws = create_autospec(AWSResources)
-    principal_acl = create_autospec(PrincipalACL)
-    aws_resource_permissions = AWSResourcePermissions(
-        installation_single_role,
-        mock_ws,
-        backend,
-        aws,
-        locations,
-        principal_acl,
-    )
-    aws_resource_permissions.create_uc_roles_cli()
-    aws.create_uc_role.assert_called_with('UC_ROLE')
-    assert (
-        aws.put_role_policy.assert_called_with(
-            'UC_ROLE', 'UC_POLICY', {'s3://BUCKET1/FOLDER1', 's3://BUCKET2/FOLDER2'}, None, None
-        )
-        is None
+    aws_resource_permissions = AWSResourcePermissions(installation_single_role, mock_ws, backend, aws, locations, "ucx")
+    role_creation = IamRoleCreation(installation_single_role, mock_ws, aws_resource_permissions)
+    role_creation.run(MockPrompts({"Above *": "yes"}))
+    aws.list_all_uc_roles.return_value = []
+    assert not aws.create_uc_role.assert_called_with('UC_ROLE')
```

Comment:
```suggestion
    aws.create_uc_role.assert_not_called_with('UC_ROLE')
```
We have linters for this